.github/workflows/publish-examples.yaml

```diff
@@ -0,0 +1,28 @@
+# trigger this on release publish in future
+#on:
+#  release:
+#    types: [published]
```

Comment:
in future, this workflow would be triggered by publishing a release in the main repo (Worklytics/psoxy), which should generate updates to the example repos based on it, to reference that latest release.